.github/workflows/dependabot-rebase.yml

```
@@ -43,6 +43,10 @@ concurrency:
   group: dependabot-update-and-merge
   cancel-in-progress: false
 
+concurrency:
```

actionlint (1.7.12)

[error] key "concurrency" is duplicated in "workflow" section. previously defined at line:42,col:1

(syntax-check)